fastai/text/models/core.py

```diff
@@ -42,7 +48,12 @@ def reset(self):
         for c in self.children(): getattr(c, 'reset', noop)()
 
 # Cell
-def get_language_model(arch, vocab_sz, config=None, drop_mult=1.):
+def get_language_model(
+    arch:callable, # Function or class that can generate a language model architecture
```

Comment:
```suggestion
    arch, # Function or class that can generate a language model architecture
```
After talking with Jeremy, let's get rid of this type hint. The docment says enough